### DISCUSSION TITLE
Add backend latency in K8s

### DISCUSSION THREAD

**Discussion by Sumudu-Sahan** [mockoon/mockoon#1632] [Help] [answered]
Hi Team,

Is there a way to add latency for a route in mockoon deployed in K8s? When exporting the openAPI file from the mockoon even after adding the latency, I cannot see the configured latency property in the exported openAPI JSON file.

**Mockoon setup**

<img width="2056" alt="Screenshot 2024-12-20 at 12 47 03 PM" src="https://github.com/user-attachments/assets/f79b6f68-06d5-4e87-919a-90a078a474ad" />

**Exported JSON**
```
{"openapi":"3.0.0","info":{"title":"Demo API","version":"1.0.0"},"servers":[{"url":"http://localhost:3000/"}],"paths":{"/username":{"get":{"description":"","responses":{"200":{"description":"","content":{"application/json":{"example":"{\"test\":\"suza\"}"}},"headers":{"":{"schema":{"type":"string"},"example":""}}}}}}}}
```

Highly appreciate your input to configure this on K8s. (Tried this on v9.1.0)

Thanks and Regards,
Sumudu W.

**Comment by 255kb** [comment, marked as answer]
Hi, you don't need to export to use your setup with the CLI/Docker. You have to directly use the [data file](https://mockoon.com/docs/latest/mockoon-data-files/data-files-location/). 
Also, I suggest you read this: https://mockoon.com/docs/latest/openapi/openapi-specification-compatibility/
OpenAPI doesn't support information like latency, rules, etc.

**Reply by Sumudu-Sahan** [reply]
Hi @255kb,

Thank you for the provided input regarding this and yes, this saved my day!!!. Hence, I'm closing the discussion.

Thanks and Regards,
Sumudu W.
